scipy/special/orthogonal.py

```diff
@@ -525,12 +532,277 @@ def h_roots(n, mu=False):
     if n < 1 or n != m:
         raise ValueError("n must be a positive integer.")
 
-    mu0 = np.sqrt(np.pi)
-    an_func = lambda k: 0.0*k
-    bn_func = lambda k: np.sqrt(k/2.0)
-    f = cephes.eval_hermite
-    df = lambda n, x: 2.0 * n * cephes.eval_hermite(n-1, x)
-    return _gen_roots_and_weights(m, mu0, an_func, bn_func, f, df, True, mu)
+    if n <= 200:
+        mu0 = np.sqrt(np.pi)
+        an_func = lambda k: 0.0*k
+        bn_func = lambda k: np.sqrt(k/2.0)
+        f = cephes.eval_hermite
+        df = lambda n, x: 2.0 * n * cephes.eval_hermite(n-1, x)
+        return _gen_roots_and_weights(m, mu0, an_func, bn_func, f, df, True, mu)
+    else:
+        nodes, weights = h_roots_asy(n)
+        if mu:
+            return nodes, weights, sum(weights)
+        else:
+            return nodes, weights
+
+
+def compute_tauk(n, k, steps=5):
+    """Helper function for lemma 3.1
```

Comment:
This comment doesn't tell me anything. I don't know that every helper function needs a full docstring in the usual style, but it ought to have something at least moderately helpful.